src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_autoscale_util.py

```diff
@@ -0,0 +1,664 @@
+    },
+    {
+        "name": "Ulaanbaatar Standard Time",
+        "offset": "+08:00"
+    },
+    {
+        "name": "North Korea Standard Time",
+        "offset": "+08:30"
+    },
+    {
+        "name": "Aus Central W. Standard Time",
+        "offset": "+08:45"
+    },
+    {
+        "name": "Transbaikal Standard Time",
+        "offset": "+09:00"
+    },
+    {
+        "name": "Tokyo Standard Time",
+        "offset": "+09:00"
+    },
+    {
+        "name": "Korea Standard Time",
+        "offset": "+09:00"
+    },
+    {
+        "name": "Russia TZ 8 Standard Time",
+        "offset": "+09:00"
+    },
+    {
+        "name": "Cen. Australia Standard Time",
+        "offset": "+09:30"
+    },
+    {
+        "name": "AUS Central Standard Time",
+        "offset": "+09:30"
+    },
+    {
+        "name": "E. Australia Standard Time",
+        "offset": "+10:00"
+    },
+    {
+        "name": "AUS Eastern Standard Time",
+        "offset": "+10:00"
+    },
+    {
+        "name": "West Pacific Standard Time",
+        "offset": "+10:00"
+    },
+    {
+        "name": "Tasmania Standard Time",
+        "offset": "+10:00"
+    },
+    {
+        "name": "Russia TZ 9 Standard Time",
+        "offset": "+10:00"
+    },
+    {
+        "name": "Lord Howe Standard Time",
+        "offset": "+10:30"
+    },
+    {
+        "name": "Bougainville Standard Time",
+        "offset": "+11:00"
+    },
+    {
+        "name": "Russia TZ 10 Standard Time",
+        "offset": "+11:00"
+    },
+    {
+        "name": "Magadan Standard Time",
+        "offset": "+11:00"
+    },
+    {
+        "name": "Norfolk Standard Time",
+        "offset": "+11:00"
+    },
+    {
+        "name": "Sakhalin Standard Time",
+        "offset": "+11:00"
+    },
+    {
+        "name": "Central Pacific Standard Time",
+        "offset": "+11:00"
+    },
+    {
+        "name": "Russia TZ 11 Standard Time",
+        "offset": "+12:00"
+    },
+    {
+        "name": "New Zealand Standard Time",
+        "offset": "+12:00"
+    },
+    {
+        "name": "UTC+12",
+        "offset": "+12:00"
+    },
+    {
+        "name": "Fiji Standard Time",
+        "offset": "+12:00"
+    },
+    {
+        "name": "Kamchatka Standard Time",
+        "offset": "+12:00"
+    },
+    {
+        "name": "Chatham Islands Standard Time",
+        "offset": "+12:45"
+    },
+    {
+        "name": "UTC+13",
+        "offset": "+13:00"
+    },
+    {
+        "name": "Tonga Standard Time",
+        "offset": "+13:00"
+    },
+    {
+        "name": "Samoa Standard Time",
+        "offset": "+13:00"
+    },
+    {
+        "name": "Line Islands Standard Time",
+        "offset": "+14:00"
+    }
+]
+
+
+def build_autoscale_schedule(autoscale_settings):
+    """ Builds up a logical model of the autoscale weekly schedule. This then has to later be
+        translated into objects that work with the Monitor autoscale API. """
+    from datetime import time
+    import json
```

Comment:
I think we can leave the `from datatime import time` and `import json` at the beginning of the file since they're lightweight and import everywhere else.